### PR TITLE
Remove indexing submodule

### PR DESCRIPTION
**Pull request by mberz**
The indexing submodule contains only three functions, ACN indexing functions are already part of the spherical submodule. I've moved the content of the indexing submodule to the spherical submodule as well.

### CODE REVIEW

File: spharpy/spherical.py [hunk 1]
```diff
@@ -509,3 +509,109 @@ def radiation_from_sphere(
...
+            sid_m[idx_n + 2*m-1] = n-m+1
+            sid_m[idx_n + 2*m] = -(n-m+1)
+        sid_m[idx_n + 2*n + 1] = 0
+        idx_n += 2*n+1
+
+    return sid_n, sid_m
+
+
+def sid2acn(n_max):
+    """Convert from SID channel indexing as proposed by Daniel.
```
Comment (ahms5) [inline spharpy/spherical.py:545]: Who is Daniel, do we know him?

File: tests/test_indexing.py [hunk 2]
```diff
@@ -1,33 +1,34 @@
 """ Tests for sh channel indexing"""
```
Comment (ahms5) [inline tests/test_indexing.py:1]: should be put these tests to ``test_spherical``

**Review by ahms5 — CHANGES_REQUESTED** [review]
thanks, just small comments

File: spharpy/spherical.py (hunk 1 above)
Comment (mberz) [reply spharpy/spherical.py]: I've added a reference to his thesis

File: tests/test_indexing.py (hunk 2 above)
Comment (mberz) [reply tests/test_indexing.py:1]: I've moved all tests for indexing (that is also acn related things) to `test_indexing`, to make this a bit more consistent

**Pull request merged**